gemrb/GUIScripts/pst/NewLife.py

```diff
@@ -250,17 +250,15 @@ def AcceptPress():
 
 	#don't add con bonus, it will be calculated by the game
 	#interestingly enough, the game adds only one level's con bonus
-	Con = Stats[4]
-	if Con > 14:
-		x = 30
-	else:
-		x = 20 + (Con-9)*2
+	Table = GemRB.LoadTable ("hpinit")
+	x = Table.GetValue (Stats[4]-1, 0)
```

Comment:
It's clearer what's going on if you use string addressing (rowname/colname), so something like `Table.GetValue (str(Stats[4]), "HP")`.

Reply:
Should be fixed now.